cpp/oneapi/dal/algo/svm/backend/cpu/train_cls_kernel.cpp

```diff
@@ -41,12 +42,19 @@ using descriptor_t = detail::descriptor_base<task::classification>;
 
 namespace daal_svm = daal::algorithms::svm;
 namespace daal_classifier = daal::algorithms::classifier;
+namespace daal_multiclass = daal::algorithms::multi_class_classifier;
+namespace daal_multiclass_internal = daal_multiclass::internal;
+
 namespace interop = dal::backend::interop;
 
 template <typename Float, daal::CpuType Cpu, typename Method>
 using daal_svm_kernel_t =
     daal_svm::training::internal::SVMTrainImpl<to_daal_method<Method>::value, Float, Cpu>;
 
+template <typename Float, daal::CpuType Cpu>
+using daal_multiclass_kernel_t = daal_multiclass::training::internal::
+    MultiClassClassifierTrainKernel<daal_multiclass::training::oneAgainstOne, Float, Cpu>;
+
 template <typename Float, typename Method>
 static result_t call_daal_kernel(const context_cpu& ctx,
```

Comment:
Suggest to think how to split this function, now it's ~120 lines of code.

Reply:
done